backend/app/api/endpoints/installed_plugins.py

```
@@ -884,7 +884,7 @@ def _ensure_publish_allowed(current_user: User) -> None:
 def init_plugin_submission(
     request: PluginSubmissionInitRequest,
     db: Session = Depends(get_db),
-    current_user: User = Depends(security.get_current_user),
+    current_user: User = Depends(security.get_current_user_jwt_apikey_tasktoken),
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)